runners/console/index.ts

```diff
@@ -346,6 +346,37 @@ export class Console extends Runner {
         return result;
     }
 
+    runExecuteCommand(runCommand: RunCommand): RunResult {
```

Comment:
Each run method should have an assert method where you check if the command was executed correctly.